wrc/parse/lexer.py

```diff
@@ -108,6 +108,11 @@ def t_PARBREAK(self, token):
         token.lexer.lineno += len(token.value)
         return token
 
+    def t_trailingwhitespace(self, token):
```

Comment:
Right now:

> [10:57] daisy:~/Documents/cube/wca/wca-compiler (illegal-whitespace|…)$ python -m wrc.wrc -o build --target=html ../wca-regulations
Error: trailing whitespace at line 19 in text 'Original source of the WCA regulations: [www.worldcubeassociation.org/regulations](https://www.worldcubeassociation.org/regulations/)    '
Couldn't compile file, the following occured:
 - Error: Couldn't build AST.


For fun here is the output without this error token:
> [10:59] daisy:~/Documents/cube/wca/wca-compiler (illegal-whitespace|✚1…)$ python -m wrc.wrc -o build --target=html ../wca-regulations
Illegal character 'O' at line 19
Illegal character 'r' at line 19
Illegal character 'i' at line 19
Illegal character 'g' at line 19
Illegal character 'i' at line 19
Illegal character 'n' at line 19
Illegal character 'a' at line 19
Illegal character 'l' at line 19
[...]